helix-term/src/keymap.rs

```diff
@@ -307,6 +316,12 @@ impl Keymap {
             Some(&KeyTrie::Leaf(cmd)) => {
                 return KeymapResult::new(KeymapResultKind::Matched(cmd), self.sticky())
             }
+            Some(&KeyTrie::Sequence(ref cmds)) => {
+                return KeymapResult::new(
+                    KeymapResultKind::MatchedSequence(cmds.clone()),
```

Comment:
Because `trie_node` is owned, I am no longer able to use a `&[Command]` for the `MatchedSequence` variant of `KeymapResultKind`. However, this owned data seems to be cloned from borrowed data, so would I be able to borrow the original borrowed data and store it in a returned `KeymapResultKind::MatchedSequence`?

Reply:
Well, it is just a `Vec<Command>` so I suppose it could be OK to just stick with cloning it?

Reply:
I think it's fine to clone it, since the vast majority of keymaps won't be a sequence anyway.

Reply:
OK